File: cpp/2108.py
import sys

n = int(input)
listlist = [int(sys.stdin.readlin()) for _ in range(n)]

#print sum
sum = 0
for num in listlist:
    sum += num
print(round(sum/n))

#print mid
sorted_list = sorted(listlist)
print(sorted_list[sum // 2]) #remember, first is 0


#print rng

File: cpp/2108.cpp
#include <iostream>
#include <algorithm>
#include <vector>
#include <cmath>
using namespace std;


int main(){
    int N=0;
    int input, sum=0, mid=0, mode=0, range=0;
    double avr=0;
    int number[8001] = {0,};
    cin >> N;
    cout << sum;
    vector<int> arr(N);

    for(int i=0 ; i<N ; i++){
        cin >> input;
        number[input+4000]++;
        sum +=0;
        arr.push_back(input);
    }

    sort(arr.begin(), arr.end()); 
    avr = sum/N;
    avr = round(avr * 10) / 10;
    mid = arr[N/2];
    range = *arr.end() - *arr.begin();



    




}